test/controllers/update/update.manage.trusts.review.legal.entities.controller.spec.ts

```diff
@@ -203,6 +203,38 @@ describe('Update - Manage Trusts - Review legal entities', () => {
       expect(resp.text).toContain('Beneficiary');
     });
 
+    test('Page displays status as active when entity is still involved in the trust', async () => {
```

Comment:
Please change name from 'entity' to 'legal entity trustee' or 'corporate trustee'. Below also.

Reply:
Renamed